eng/docgeneration/Generate-DocIndex.ps1

```diff
@@ -1,8 +1,23 @@
+# Generates an index page for cataloging different versions of the Docs
 [CmdletBinding()]
 Param (
     $RepoRoot,
-    $DocGenDir
+    $DocGenDir,
+    $lang = "java",
+    $packageNameRegex = "",
+    $packageNameReplacement = ""
 )
+. (Join-Path $PSScriptRoot ../../eng/common/scripts/Package-Properties.ps1)
+
+# A mapping is used to fill in 1. Title of home page, 2. The azure storage link
+$LangMapping = @{
```

Comment:
We should be importing the common.ps1 file instead of the Package-Properties.ps1 directly. Once you import Common you should be able to use $languageShort (https://github.com/Azure/azure-sdk-for-js/blob/master/eng/scripts/Language-Settings.ps1#L2).